### ISSUE TITLE
Add to NPM registry

### ISSUE THREAD

**Issue by ndcunningham** [OhScee/confetti-popping#3]
It would be great if we could just add this behind a npm registry so that we could easily install it.

**Comment by OhScee** [comment]
I agree. Unfortunately, I've had a lot of issues trying to get the latest angular-cli library format to work with npm properly. Maybe they've posted an update to resolve this, but I won't have proper time to investigate it for a bit

**Comment by ndcunningham** [comment]
yea i think there was a problem with `ng-packagr` and the CLI.
I don't remember what i did to solve this i could look into it though if that's fine.

**Comment by OhScee** [comment]
Yeah absolutely. I did a bit of investigation myself and I think it's largely @angular/core... and cli version related. Confetti is only anticipating a version of ~5, and I think later versions of angular and angular-cli use a different tree shaking method, so it's messing with the library

**Comment by ndcunningham** [comment]
Yea that's correct the build process post ng5 was changed and as such `@angular/core` and friends were updated.

**Comment by OhScee** [comment]
hahaha @angular and friends, love it. But yeah, I'm hoping it's a simple version adjustment fix

**Comment by ndcunningham** [comment]
Yea same! Will update after i make a few adjustments to your fireworks func making rxjs friendly

**Comment by OhScee** [comment]
Very cool! Looking forward to the pr :)
